Command: /home/mati/.espressif/python_env/idf6.0_py3.13_env/bin/python /home/mati/esp/esp-idf/tools/idf_monitor.py -p /dev/ttyACM0 -b 115200 --toolchain-prefix riscv32-esp-elf- --target esp32h2 --revision 0 --decode-panic backtrace /home/mati/esp/esp-zigbee-sdk/examples/esp_zigbee_HA_sample/HA_thermostat/build/thermostat.elf /home/mati/esp/esp-zigbee-sdk/examples/esp_zigbee_HA_sample/HA_thermostat/build/bootloader/bootloader.elf -m '/home/mati/.espressif/python_env/idf6.0_py3.13_env/bin/python' '/home/mati/esp/esp-idf/tools/idf.py' '-p' '/dev/ttyACM0'
[0;33m--- esp-idf-monitor 1.6.2 on /dev/ttyACM0 115200[0m
[0;33m--- Quit: Ctrl+] | Menu: Ctrl+T | Help: Ctrl+T followed by Ctrl+H[0m
[0;33m--- 0x40011bdc: uart_hal_get_txfifo_count in ROM
[0m
[0;33m--- 0x4083c2d0: esp_bootloader_get_description at /home/mati/esp/esp-idf/components/esp_bootloader_format/esp_bootloader_desc.c:39 (discriminator 1)
[0m
[0;33m--- 0x4083efd0: esp_flash_encryption_enabled at /home/mati/esp/esp-idf/components/bootloader_support/src/flash_encrypt.c:89
[0m
[0;33m--- 0x4083c2da: call_start_cpu0 at /home/mati/esp/esp-idf/components/bootloader/subproject/main/bootloader_start.c:25
[0m
[0;33m--- Stack dump detected[0m
[0;33m--- 0x4200cede: esp_app_zb_attribute_handler at /home/mati/esp/esp-zigbee-sdk/examples/esp_zigbee_HA_sample/HA_thermostat/main/esp_zb_thermostat.c:348
--- 0x4200d032: zb_attribute_reporting_handler at /home/mati/esp/esp-zigbee-sdk/examples/esp_zigbee_HA_sample/HA_thermostat/main/esp_zb_thermostat.c:379
[0m
[0;33m--- 0x4001117a: __retarget_lock_release_recursive in ROM
[0m
[0;33m--- 0x40800001: _vector_table at /home/mati/esp/esp-idf/components/riscv/vectors_intc.S:54
[0m
[0;33m
--- Backtrace:

[0m
[0;33m--- 0x400031b6: software_reset_cpu in ROM
[0m
[0;33m--- 0x4083c2d0: esp_bootloader_get_description at /home/mati/esp/esp-idf/components/esp_bootloader_format/esp_bootloader_desc.c:39 (discriminator 1)
[0m
[0;33m--- 0x4083efd0: esp_flash_encryption_enabled at /home/mati/esp/esp-idf/components/bootloader_support/src/flash_encrypt.c:89
[0m
[0;33m--- 0x4083c2da: call_start_cpu0 at /home/mati/esp/esp-idf/components/bootloader/subproject/main/bootloader_start.c:25
[0m
[0;33m--- Stack dump detected[0m
[0;33m--- 0x4200cede: esp_app_zb_attribute_handler at /home/mati/esp/esp-zigbee-sdk/examples/esp_zigbee_HA_sample/HA_thermostat/main/esp_zb_thermostat.c:348
--- 0x4200d032: zb_attribute_reporting_handler at /home/mati/esp/esp-zigbee-sdk/examples/esp_zigbee_HA_sample/HA_thermostat/main/esp_zb_thermostat.c:379
[0m
[0;33m--- 0x4001117a: __retarget_lock_release_recursive in ROM
[0m
[0;33m--- 0x40800001: _vector_table at /home/mati/esp/esp-idf/components/riscv/vectors_intc.S:54
[0m
[0;33m--- Warning: Failed to run gdb_panic_server.py script: Command '['riscv32-esp-elf-gdb', '--batch', '-n', '/home/mati/esp/esp-zigbee-sdk/examples/esp_zigbee_HA_sample/HA_thermostat/build/thermostat.elf', '-ex', 'add-symbol-file /home/mati/esp/esp-zigbee-sdk/examples/esp_zigbee_HA_sample/HA_thermostat/build/bootloader/bootloader.elf', '-ex', 'target remote | "/home/mati/.espressif/python_env/idf6.0_py3.13_env/bin/python" -m esp_idf_panic_decoder --target esp32h2 "/tmp/tmpdy8tbfvu"', '-ex', 'bt']' returned non-zero exit status 1.
--- Warning: b'add symbol table from file "/home/mati/esp/esp-zigbee-sdk/examples/esp_zigbee_HA_sample/HA_thermostat/build/bootloader/bootloader.elf"\nRemote communication error.  Target disconnected: error while reading: Connection reset by peer.\nNo stack.\n'

[0m
[0;33m--- 0x400031b6: software_reset_cpu in ROM
[0m
[0;33m--- 0x4083c2d0: esp_bootloader_get_description at /home/mati/esp/esp-idf/components/esp_bootloader_format/esp_bootloader_desc.c:39 (discriminator 1)
[0m
[0;33m--- 0x4083efd0: esp_flash_encryption_enabled at /home/mati/esp/esp-idf/components/bootloader_support/src/flash_encrypt.c:89
[0m
[0;33m--- 0x4083c2da: call_start_cpu0 at /home/mati/esp/esp-idf/components/bootloader/subproject/main/bootloader_start.c:25
[0m
[0;33m--- Stack dump detected[0m
[0;33m--- 0x4200cede: esp_app_zb_attribute_handler at /home/mati/esp/esp-zigbee-sdk/examples/esp_zigbee_HA_sample/HA_thermostat/main/esp_zb_thermostat.c:348
--- 0x4200d032: zb_attribute_reporting_handler at /home/mati/esp/esp-zigbee-sdk/examples/esp_zigbee_HA_sample/HA_thermostat/main/esp_zb_thermostat.c:379
[0m
[0;33m--- 0x4001117a: __retarget_lock_release_recursive in ROM
[0m
[0;33m--- 0x40800001: _vector_table at /home/mati/esp/esp-idf/components/riscv/vectors_intc.S:54
[0m
[0;33m--- Warning: Failed to run gdb_panic_server.py script: Command '['riscv32-esp-elf-gdb', '--batch', '-n', '/home/mati/esp/esp-zigbee-sdk/examples/esp_zigbee_HA_sample/HA_thermostat/build/thermostat.elf', '-ex', 'add-symbol-file /home/mati/esp/esp-zigbee-sdk/examples/esp_zigbee_HA_sample/HA_thermostat/build/bootloader/bootloader.elf', '-ex', 'target remote | "/home/mati/.espressif/python_env/idf6.0_py3.13_env/bin/python" -m esp_idf_panic_decoder --target esp32h2 "/tmp/tmp0b76u6e2"', '-ex', 'bt']' returned non-zero exit status 1.
--- Warning: b'add symbol table from file "/home/mati/esp/esp-zigbee-sdk/examples/esp_zigbee_HA_sample/HA_thermostat/build/bootloader/bootloader.elf"\nRemote communication error.  Target disconnected: error while reading: Connection reset by peer.\nNo stack.\n'

[0m
[0;33m--- 0x400031b6: software_reset_cpu in ROM
[0m
[0;33m--- 0x4083c2d0: esp_bootloader_get_description at /home/mati/esp/esp-idf/components/esp_bootloader_format/esp_bootloader_desc.c:39 (discriminator 1)
[0m
[0;33m--- 0x4083efd0: esp_flash_encryption_enabled at /home/mati/esp/esp-idf/components/bootloader_support/src/flash_encrypt.c:89
[0m
[0;33m--- 0x4083c2da: call_start_cpu0 at /home/mati/esp/esp-idf/components/bootloader/subproject/main/bootloader_start.c:25
[0m
[0m
